cmd/build-init/main.go

```diff
@@ -59,7 +61,6 @@ const (
 	platformDir               = "/platform"
 	buildSecretsDir           = "/var/build-secrets"
 	imagePullSecretsDir       = "/imagePullSecrets"
```

Comment:
the secrets contained in this dir could conflict with the secrets coming from the Build's service account, maybe we should deduplicate them.